src/backends/interflop-ieee/interflop_ieee.c

```diff
@@ -327,6 +335,9 @@ static void _interflop_cmp_double(const enum FCMP_PREDICATE p, const double a,
   debug_print_double(context, COMPARISON, str, a, b, *c);
 }
 
+// Include vector operations
+#include "interflop_ieee_vector.h"
+
```

Comment:
In vector files I use macro that are defined on the original file.

Reply:
I think I can put this `.h` file in a `.c` file instead and it will fix the problem.

Reply:
Actually, I don't know if it will fix the problem because we will still need other macro. I don't know.